Focus/LensOperators.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Operator.swift
+//  Focus
+//
+//  Created by Robert Widmann on 7/2/15.
+//  Copyright © 2015 TypeLift. All rights reserved.
+//
+
+/// Compose | Applies one function to the result of another function to produce a third function.
+infix operator • {
+    associativity right
```

Comment:
I'm worried about this entire file conflicting with Swiftz if they are used together